src/index.js

```diff
@@ -7,16 +7,23 @@ import { Pond } from '@actyx-contrib/react-pond';
 function Waiting() {
   return (
     <div className="loading">
-      Waiting for connection to ActyxOS on localhost — is it running?
+      Waiting for connection to Actyx on localhost — is it running?
       <br />
-      You may need to reload after starting ActyxOS.
+      You may need to reload after starting Actyx.
     </div>
-  )
+  );
 }
 
 ReactDOM.render(
   <React.StrictMode>
-    <Pond loadComponent={<Waiting />}>
+    <Pond
+      loadComponent={<Waiting />}
```

Comment:
We just made `onError` required in the latest `react-pond`.
Let's add a reloading hander here like in https://github.com/Actyx/advanced-example/blob/master/src/order-management/root.tsx#L7